cabal.project

```diff
@@ -26,6 +27,9 @@ package ghcide
 
 write-ghc-environment-files: never
 
-index-state: 2020-10-16T04:00:00Z
+index-state: 2020-10-16T03:50:06Z
 
 allow-newer: data-tree-print:base
+
+-- To ensure the build get the version with the fix for #498
+constraints: implicit-hie >= 0.1.2.0
```

Comment:
shouldn't this be in the cabal file?

Reply:
well, the dependency is hls -> ghcide -> implicit-hie-cradle -> implicit-hie
implicit-hie-cradle has no bounds on implicit-hie so everybody should get the last version *if you has executed a cabal update* before the build

So imo the contraint should be in implicit-hie-cradle cabal file, making an hackage revision and not here but at same time i want to force hls users to get the version with the fix using `cabal update` so :man_shrugging:

Reply:
It should be removed when we use a implicit-hie-cradle that ensures we get the correct implicit-version, maybe i should comment that

Reply:
@fendor comment added